charts/jenkins/README.md

```diff
@@ -415,6 +416,41 @@ $ helm install my-release -f values.yaml stable/jenkins
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+#### Defining Caches for your agent pods
+
+Its possible to define a set of cache volumes that can be mounted by agent pods for storing libraries and other artifacts for build.
+- The volume defined, under `pvc`, will be created by the chart.
+- The volume has to be mounted under the `volumes` section.
+- Along with the cache, a `CronJob` can be defined under the `clearJob` section to perform cache activities, including clearing old libraries, on a periodic basis.
+- The `CronJob` is required both to prevent unused libraries from accumulating over time and to avoid security issues.
+
```

Comment:
Should we add one working example here?
- defining a cache
- a job which is using that cache (might be something as simple as `echo "writing files which should be cached" > /mycache`
- a CRON job which cleans things up

Reply:
Thank  you Aaron, I have merged the changes. I can see that apart from no inheritance, there is also auto mounting of the created PVC volumes. 

@torstenwalter, will test with these changes and create a commit with a simple example this week.